Apps/LabRecorder/CMakeLists.txt

```diff
@@ -1,223 +1,87 @@
 # CMake version
-cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.5)
+
+project(LabRecorder
+	LANGUAGES CXX
+	VERSION 0.1.0)
+
+# load LSLAppBoilerplate if not done already
+if(NOT TARGET LSL::lsl)
+	if(NOT LSL_ROOT)
+		message(FATAL_ERROR "When building a single app out of tree you need to set LSL_ROOT ( cmake -DLSL_ROOT=/path/to/installed/lsl)")
+	endif()
+	list(APPEND CMAKE_MODULE_PATH "${LSL_ROOT}/cmake")
+	include(LSLAppBoilerplate)
+endif()
 
-##################
 # GENERAL CONFIG #
-##################
-
-# Include cmake modules
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
-
-# Set policies
 cmake_policy(SET CMP0028 NEW) # ENABLE CMP0028: Double colon in target name means ALIAS or IMPORTED target.
 cmake_policy(SET CMP0054 NEW) # ENABLE CMP0054: Only interpret if() arguments as variables or keywords when unquoted.
-cmake_policy(SET CMP0042 NEW) # ENABLE CMP0042: MACOSX_RPATH is enabled by default.
 cmake_policy(SET CMP0063 NEW) # ENABLE CMP0063: Honor visibility properties for all target types.
 
 # Meta information about the project
-set(META_PROJECT_NAME        "LabRecorder")
```

Comment:
Most of this gets set automatically in LSLBoilerplate and the `project()` declaration at the top